### PR TITLE
Update pages

### PR DESCRIPTION
**Pull request by kbdharun**
Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

## Changes

- Fix the Image in the latest guide.
- Update Steam, Flatpak, Driver Installation and Distrobox guides.
- Add Installing Drivers as DEBs Section.

### CODE REVIEW

File: _posts/2023-01-07-install-steam.md [hunk 1]
```diff
@@ -10,48 +10,41 @@ published: true
...
-project, which allows you to run Windows games on Linux. This is a great feature
-for Linux users, because it extends the number of games available on the platform
-by a lot. Take a look at [ProtonDB](https://www.protondb.com) to see how well
-your favorite games work with Proton.
+- On 21 August 2018, Steam introduced the [Proton](https://github.com/ValveSoftware/Proton) project, which allows you to run Windows games on Linux. It is an essential feature for Linux gamers as it extends the number of games available on the platform. Check out [**ProtonDB**](https://www.protondb.com) to see how well your favourite games work with Proton.
 
 ## Install Steam
 
-In Vanilla OS there are many ways to install software. The easiest way is to use
+In Vanilla OS there are many ways to install applications. The easiest way is to use
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:19]:
```suggestion
In Vanilla OS there are many ways to install Steam. The easiest way is to use
```

File: _posts/2023-01-07-install-steam.md [hunk 2]
```diff
@@ -10,48 +10,41 @@ published: true
...
-[Steam](https://steampowered.com) is the most popular gaming platform. It allows
-you to download and play a large number of games, and it's available for Linux.
+- [**Steam**](https://steampowered.com) is the most popular gaming platform. It allows you to download and play a vast library of games. It's client is available natively on Linux.
 
-In 21 August 2018, Steam has introduced the [Proton](https://github.com/ValveSoftware/Proton)
-project, which allows you to run Windows games on Linux. This is a great feature
-for Linux users, because it extends the number of games available on the platform
-by a lot. Take a look at [ProtonDB](https://www.protondb.com) to see how well
-your favorite games work with Proton.
+- On 21 August 2018, Steam introduced the [Proton](https://github.com/ValveSoftware/Proton) project, which allows you to run Windows games on Linux. It is an essential feature for Linux gamers as it extends the number of games available on the platform. Check out [**ProtonDB**](https://www.protondb.com) to see how well your favourite games work with Proton.
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:15]:
```suggestion
- On 21 August 2018, Steam introduced the [Proton](https://github.com/ValveSoftware/Proton) project, which allows you to run Windows games on Linux. It is an essential tool for Linux gamers as it extends the number of games available on the platform. Check out [**ProtonDB**](https://www.protondb.com) to see how well your favourite games work with Proton.
```

File: _posts/2023-01-07-install-steam.md [hunk 3]
```diff
@@ -10,48 +10,41 @@ published: true
...
+flatpak install flathub com.valvesoftware.Steam
 ```
 
 ### Apx
 
-Apx is a package manager which installs software in containers thanks to
-[Distrobox](https://github.com/89luca89/distrobox). In our tests, Fedora is the
-best distro to run Steam in a container and it is highly recommended to use it.
+**Apx** is a package manager which installs software in containers thanks to
+[Distrobox](https://github.com/89luca89/distrobox). In our tests, Fedora is the best distro to run Steam in a container . We highly recommend using it to install Steam.
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:39]:
```suggestion
[Distrobox](https://github.com/89luca89/distrobox). In our tests, Fedora is the best distro to run Steam in a container. We highly recommend using it to install Steam.
```

File: _posts/2023-01-07-install-steam.md [hunk 4]
```diff
@@ -75,37 +68,39 @@ apx --dnf install xorg-x11-drv-amdgpu mesa-libGL.i686 mesa-dri-drivers.i686
...
+- If the icon doesn't appear in the applications menu, you can add it manually using the following command:
 
 ```bash
 apx --dnf export steam
 ```
 
 #### Ubuntu container
 
-On Ubuntu, we need to add the `multiverse` repository to our sources list:
+- On Ubuntu, we need to add the `multiverse` repository to our sources list using the follow:
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:90]:
```suggestion
- On Ubuntu, we need to add the `multiverse` repository to our sources list using the following command:
```

File: _posts/2023-01-07-install-steam.md [hunk 5]
```diff
@@ -75,37 +68,39 @@ apx --dnf install xorg-x11-drv-amdgpu mesa-libGL.i686 mesa-dri-drivers.i686
 apx --dnf xorg-x11-drv-intel mesa-libGL.i686 mesa-dri-drivers.i686
 ```
 
-Now we can install Steam:
+- Now we can install Steam using the following command:
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:71]:
```suggestion
- We can now install Steam using the following command:
```

File: _posts/2023-01-07-install-steam.md [hunk 6]
```diff
@@ -117,36 +112,34 @@ which is `525.60.11` in my case. Then we need to install the drivers for the con
...
 
-- **AMD/Intel**
+- **AMD/Intel**:-
 
 ```bash
 apx --apt install mesa-utils mesa-utils-extra
 ```
 
-Now we can install Steam:
+Now we can install Steam using the following command:
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:121]:
```suggestion
- We can now install Steam using the following command:
```

File: _posts/2023-01-07-install-steam.md [hunk 7]
```diff
@@ -10,48 +10,41 @@ published: true
...
-flatpak install --user flathub com.valvesoftware.Steam
+flatpak install flathub com.valvesoftware.Steam
 ```
 
 ### Apx
 
-Apx is a package manager which installs software in containers thanks to
-[Distrobox](https://github.com/89luca89/distrobox). In our tests, Fedora is the
-best distro to run Steam in a container and it is highly recommended to use it.
+**Apx** is a package manager which installs software in containers thanks to
```
Comment (MonsterObserver) [inline _posts/2023-01-07-install-steam.md:38]:
```suggestion
**apx** is a package manager which installs software in containers thanks to
```

File: _posts/2022-12-09-install-flatpaks.md [hunk 8]
```diff
@@ -8,69 +8,57 @@ authors:
...
 
 ```bash
 apx run flatpak install flathub org.gnome.gedit
 ```
 
 ## Configure Flathub
 
-Flatpak needs to know where to look for applications. The default and most
-popular repository for installing Flatpaks is Flathub, which can be added using the following command:
+Flatpak needs to know where to look for applications. The default and most popular repository for installing Flatpaks are Flathub, which can get added using the following command:
```
Comment (MonsterObserver) [inline _posts/2022-12-09-install-flatpaks.md:38]:
```suggestion
Flatpak needs to know where to look for applications. The default and most popular repository for installing Flatpaks is Flathub, which you can add using the following command:
```

File: _posts/2022-12-10-install-additional-drivers.md [hunk 9]
```diff
@@ -1,50 +1,84 @@
...
+- If you prefer to install drivers manually, you can do it using the transactional shell. Open the terminal and run the following command:
 
 ```bash
 sudo abroot exec apt install <driver>
 ```
 
-where `<driver>` is the package name that offers the driver you want to
-install. For example, to install the NVIDIA driver at version 525, you can run:
+Where `<driver>` is the package name that offers the driver you want to
+install. For example,  Installing the NVIDIA driver at version 525 can be done by executing the following command:
```
Comment (MonsterObserver) [inline _posts/2022-12-10-install-additional-drivers.md:43]:
```suggestion
install. For example, installing the NVIDIA driver at version 525 can be done by executing the following command:
```

File: _posts/2022-12-10-install-additional-drivers.md [hunk 10]
```diff
@@ -1,50 +1,84 @@
...
-then reboot once the installation is complete.
+- After installation, exit the shell using the `exit` command and wait for the changes to be applied, then reboot once the installation is complete.
+
+### DEBs
+
+- You can install drivers present as DEB files through ABRoot using the following steps:-
+	- In a non-privileged terminal, you can copy the DEB file to `/tmp` using the following command:-
+
+```bash
+sudo cp <path/to/driver.deb> /tmp
```
Comment (MonsterObserver) [inline _posts/2022-12-10-install-additional-drivers.md:57]:
```suggestion
cp <path/to/driver.deb> /tmp
```

**Review by MonsterObserver — CHANGES_REQUESTED** [review]
A bunch of small little fixes :)

File: _posts/2022-12-09-install-flatpaks.md [hunk 11]
```diff
@@ -8,69 +8,57 @@ authors:
...
-using `apx`:
+- Alternatively, you can install Flatpak inside the Managed Container using `apx`:
 
 ```bash
 apx install flatpak
 ```
 
-in this case, you will always need to run Flatpak commands inside the Apx
-container:
+In this case, you will always need to run Flatpak commands inside the Apx container:
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply _posts/2022-12-09-install-flatpaks.md:30]:
```suggestion
In this case, you will always need to run Flatpak commands inside the `apx` container:
```

File: _posts/2022-12-09-install-flatpaks.md [hunk 12]
```diff
@@ -8,69 +8,57 @@ authors:
...
-NOTE: The `--user` flag, which tells Flatpak to install applications only for the
-current user, this is needed because the root partitions in Vanilla OS are
-small-sized and must be updated in a transactional way only.
+**_NOTE_**: The `--user` flag, which tells Flatpak to install applications only for the current user, is needed because the root partitions in Vanilla OS are small-sized and must get updated in a transactional way only.
 
 ## Install an application
 
-There are two ways to install an application: using GNOME Software or using
-the command line.
+There are two ways to install a Flatpak application: using _GNOME Software or through the command line_.
```
Comment (kbdharun) [inline _posts/2022-12-09-install-flatpaks.md:48]:
```suggestion
There are two ways to install a Flatpak applications using _GNOME Software or through the command line_.
```

File: _posts/2022-12-10-install-additional-drivers.md (hunk 10 above)
Comment (kbdharun) [reply _posts/2022-12-10-install-additional-drivers.md:57]: Isn't sudo required here?

**Pull request merged**